ui/src/screens/Post/components/NewPostForm.tsx

```diff
@@ -60,19 +60,12 @@ const NewPostForm: React.FC = () => {
   const onSubmit = (values: FormFields) => {
     const postJob = async () => {
       try {
-        const response = await JobsGoWhereApiClient.post(
+        await JobsGoWhereApiClient.post(
           `${process.env.REACT_APP_API}/${values.type}`,
           JSON.stringify(values),
-          {
-            headers: {
-              "Content-Type": "application/json",
-            },
-          },
         );
-
-        toast("Your post has been successfully created! 👍");
-        await new Promise((response) => setTimeout(response, 3000));
```

Comment:
I should probably have marked this earlier; it looks like you might have a misunderstanding here on callbacks (which I've seen you have before).

The `response` in line 63 is not being used at all (hence can be deleted).
The `response` in line 74 has nothing to do with the one on line 63. It can be `foo` and it still works. Typically, we pass a callback function into `new Promise` with 2 params: `resolve` and `reject`
```
new Promise( (resolve, reject) => { ... } )
```
You basically just named your `resolve` function as `response`. No harm done, it just looks like you might be confused, or could be just a typo. Just wanted to point it out in case it was the former.